src/gql/queries/get-patch-tasks.ts

```diff
@@ -17,7 +16,7 @@ export const GET_PATCH_TASKS = gql`
   ) {
     patchTasks(
       patchId: $patchId
-      limit: ${PATCH_TASKS_LIMIT}
+      limit: 10
```

Comment:
should probably pass limit at a variable to query instead of hardcoding in query